RP2040Home/RP2040Home.py

```diff
@@ -55,13 +60,34 @@ def start_connection(self):
                 user=self.config.mqtt_config.user,
                 password=self.config.mqtt_config.password),
             machine)
-        haMqttClient.mqttInitialise(True)
+        self.haMqttClient.mqttInitialise(True)
+        return self
+        
+    def subscribe(self):
+        def my_dicovery_callback(t):
+            self.haMqttClient.mqttHADiscoveryPost()
+
+        def my_wlan_connect_callback(t):
+            self.connect_wlan()
+
+        ten_minutes_as_ms = 10*self.minutes_to_seconds*self.seconds_to_ms
```

Comment:
We should abstract this out into something that can be driven through the config.